spinegeneric/cli/package_for_correction.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python
+#
+# Script to package data for manual correction.
+#
+# For usage, type: python package_for_correction.py -h
+#
+# Author: Julien Cohen-Adad
+
+
+import os
+import sys
+import shutil
+import tempfile
+from textwrap import dedent
+import argparse
+import yaml
+import coloredlogs
+
+import spinegeneric as sg
+import spinegeneric.utils
+import spinegeneric.bids
+
+
+def get_parser():
+    """
+    parser function
+    """
+    parser = argparse.ArgumentParser(
+        description='Package data for manual correction. In case processing is ran on a remote cluster, it it '
+                    'convenient to generate a package of the files that need correction to be able to only copy these '
+                    'files locally, instead of copying the ~20GB of total processed files.',
+        formatter_class=sg.utils.SmartFormatter,
+        prog=os.path.basename(__file__).strip('.py')
+    )
+    parser.add_argument(
+        '-config',
+        metavar=sg.utils.Metavar.file,
+        required=True,
```

Comment:
Although is this argument required, when I print help, it is listed under `optional arguments`. Maybe, it could be confusing for somebody.

<img width="703" alt="image" src="https://user-images.githubusercontent.com/39456460/87330857-71b95300-c539-11ea-8c57-a467bd98ad11.png">